silkworm/db/access_layer.cpp

```diff
@@ -37,6 +37,35 @@ std::optional<BlockHeader> read_header(lmdb::Transaction& txn, uint64_t block_nu
     return header;
 }
 
+// TG ReadTransactions
+static std::vector<Transaction> read_transactions(lmdb::Transaction& txn, uint64_t base_id, uint64_t count) {
+    std::vector<Transaction> v;
+    if (count == 0) {
+        return v;
+    }
+    v.reserve(count);
+
+    auto table{txn.open(table::kEthTx)};
+
+    Bytes txn_key(8, '\0');
+    boost::endian::store_big_u64(txn_key.data(), base_id);
+    MDB_val key_mdb{to_mdb_val(txn_key)};
+    MDB_val data_mdb;
+
+    uint64_t i{0};
+    for (int rc{table->seek_exact(&key_mdb, &data_mdb)}; rc != MDB_NOTFOUND && i < count;
+         rc = table->get_next(&key_mdb, &data_mdb), ++i) {
+        lmdb::err_handler(rc);
+        ByteView data{from_mdb_val(data_mdb)};
+
+        Transaction eth_txn;
```

Comment:
Can we have a Transaction constructor with ByteView so we can `emplace_back` directly into vector ?

Reply:
> Can we have a Transaction constructor with ByteView so we can `emplace_back` directly into vector ?

I suspect the resulting performance gain will be insignificant. Also, I'd like to avoid coupling Transaction too tightly to its RLP serialization.